homeassistant/components/matter/config_flow.py

```diff
@@ -0,0 +1,69 @@
+"""Config flow for Matter integration."""
+from __future__ import annotations
+
+from typing import Any
+
+from matter_server.client.client import Client
+from matter_server.client.exceptions import CannotConnect, InvalidServerVersion
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_URL
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import aiohttp_client
+
+from .const import DOMAIN, LOGGER
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_URL, default="http://172.30.32.1:5580/chip_ws"): str,
```

Comment:
1) Let's adjust that endpoint to just ws in the server (later/TODO).
2) Let's make sure we work towards a situation where this address is hidden to some advanced place only and it will all be auto discovered/installed.